lib/stemcell/launcher.rb

```diff
@@ -291,5 +297,44 @@ def try_file(opt="")
         File.read(File.expand_path(opt)) rescue opt
     end
 
+    MAX_ATTEMPT = 3
+    def initial_retry_sec
+      return 1
+    end
+
+    # Return a Hash of instance => error. Empty hash indicates "no error"
+    # for code block:
+    #   - if block returns nil, success
```

Comment:
Are we sure that `instance.terminate` and `instance.tags.set(tags)` above return `nil`? Should we add `nil` explicitly after these calls?